### ISSUE TITLE
Tornado Support

### ISSUE THREAD

**Issue by v3ss0n** [JeffHeard/sondra#2]
Is there any work on tornado support currently ?

**Comment by JeffHeard** [comment]
No, there's not, but I would love to see it happen. I'm actually using Tornado myself, but i've been using RethinkDB directly for that and supporting REST only over Flask. Are you interested in contributing in general? 

I'm afraid development has been rather rapid on this and documentation and release info rather scant, as I'm using it for a project and thus it's being sculpted to fit my needs.  I'd love for it to be more generally used though, and I'd support standardizing it and getting a release process put together.
